src/utils/resources.ts

```diff
@@ -0,0 +1,21 @@
+import { app } from 'electron'
+import { readFile } from 'fs/promises'
+import path from 'path'
+
+const RESOURCES_ROOT_PATH = MAIN_WINDOW_VITE_DEV_SERVER_URL
+  ? path.join(app.getAppPath(), 'resources')
+  : process.resourcesPath
+
+const RESOURCE_INDEX = {
+  'browser-script': 'browser/index.js',
```

Comment:
My idea is that we can add every resource we need to load here, e.g. the checks/group snippets.